unpackaged/config/qa/package.xml

```diff
@@ -47,6 +47,7 @@
     </types>
     <types>
         <members>Contact-Contact Lightning Layout</members>
+        <members>npe01__OppPayment__c-npe01__Payment Layout</members>
```

Comment:
Do we commit the changes to "package.xml" file as part of the retrieve_changes task ? I'll let @davidmreed confirm.

Reply:
Yeah, we need this here.